### ISSUE TITLE
Atlassian APIs being blocked by block lists

### ISSUE THREAD

**Issue by amclin** [brave/adblock-lists#752]
Brave incorrectly blocks Atlassian APIs, though I can't find anything explicitly in these lists that would do so

```commons.ce18f346ad4ccfd19a11.8.js:352 POST https://api.atlassian.com/metal/ingest net::ERR_BLOCKED_BY_CLIENT```

The domain for the pages calling this API are branded subdomains (Atlassian's client orgs), like `example.atlassian.com` but always call `api.atlassian.com` as a POST request which is getting blocked.

**Comment by ryanbr** [comment]
Is this still an issue? @amclin how can it be reproduceable and the full urls being blocked?

**Comment by amclin** [comment]
@ryanbr that is the full URL and shows up multiple times when using normal Jira pages on Atlassian.net

Presumably this can be replicated by creating a free or trial JIRA project on `Atlassian.net` and observing the browser console while using said project.

Could this be because of a cross-domain POST? The pages are hosted at `<unique account name>.atlassian.net` and the POST request goes to `api.atlassian.com`

<img width="1320" alt="Screen Shot 2022-02-07 at 9 18 18 AM" src="https://user-images.githubusercontent.com/105728/152838700-50b6f565-5214-48ea-a45c-a2f904c349d7.png">

**Comment by ryanbr** [comment]
What shows up in shields, ads/trackers @amclin

**Comment by amclin** [comment]
<img width="407" alt="Screen Shot 2022-02-10 at 12 24 33 PM" src="https://user-images.githubusercontent.com/105728/153490523-773cc785-5714-4cbf-8323-8dd2d3ba3505.png">

**Comment by ryanbr** [comment]
Has this been resolved @amclin  ?

**Comment by ryanbr** [comment]
Closing, considered resolved

**Comment by ctracy** [comment]
I believe I have run into a use case which triggers the issue described above.  Here are the details on what triggered it:

(1) create a wiki page (hosted on atlassian.net)
(2) on that wiki page, add an iframe element
(3) point the iframe URL to something like a "published" google spreadsheet

In case it matters - for our case, the "published" google spreadsheet was not generally accessible by the public — only authenticated users within our institution can access it.

If you turn shields down, everything works correctly.

I am not sure if it is best to create a new issue, or if this one could be reopened... I found this issue after seeing the URL listed under Brave Shields "Advanced Controls" and clicking the "1" which was blocked, which led me to https://api.atlassian.com/metal/ingest, and then finally to the brave-unbreak.txt file: https://raw.githubusercontent.com/brave/adblock-lists/master/brave-unbreak.txt

When I use Developer Tools in Brave and reload the page, under "Console" I am seeing the same failed POSTs mentioned by @amclin on Feb 7.
